images/ubuntu/scripts/build/Install-PowerShellAzModules.ps1

```diff
@@ -28,17 +27,16 @@ foreach ($module in $modules)
     # Get github release asset for each version
     foreach ($toolVersion in $module.zip_versions) {
         $asset = $assets | Where-Object version -eq $toolVersion `
-        | Select-Object -ExpandProperty files `
-        | Select-Object -First 1
-
-        Write-Host "Installing $($module.name) $toolVersion ..."
-        if ($null -ne $asset) {
-            Write-Host "Download $($asset.filename)"
-            wget $asset.download_url -nv --retry-connrefused --tries=10 -P $installPSModulePath
-        } else {
-            Write-Host "Asset was not found in versions manifest"
+            | Select-Object -ExpandProperty files `
+            | Select-Object -First 1
+
+        if (-not $asset) {
+            Write-Host "Asset for ${moduleName} ${toolVersion} was not found in versions manifest"
             exit 1
         }
+
+        Write-Host "Downloading asset for ${moduleName} ${toolVersion}: $($asset.filename)"
+        wget $asset.download_url -nv --retry-connrefused --tries=10 -P $installPSModulePath
```

Comment:
Should we use PowerShell native here instead of `wget`? E.g. `Invoke-RestMethod`